frontend/src/components/loading/index.css

```diff
@@ -0,0 +1,86 @@
+.loading-wrapper {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.loading-spinner {
+  color: official;
```

Comment:
What does `official` means here ?

Reply:
In my opinion, official just like inherit.

Reply:
> In my opinion, official just like inherit.

There is no such `official` option for CSS `color` property, are you sure with your comment?